uint64 id
string name
int8 type
int8 PYTHON=1
int8 PUPPET=2
string owner
